Sources/Yams/Constructor.swift

```diff
@@ -54,6 +54,8 @@ public final class Constructor {
                 return result
             }
             return [Any].construct_seq(from: sequence)
+        case .unresolved:
+            fatalError("Unable to get Swift value from unresolved node type.")
```

Comment:
This is what I'm troubleshooting now. Once this is fixed, I should be able to make a PR against YAMS.